test/rpc_test.py

```diff
@@ -660,104 +660,96 @@ def _test_multi_remote_call(self, fn, args_fn=lambda x: (), kwargs_fn=lambda x:
             self.assertEqual(rrefs[i].to_here(), expected[i])
 
     @dist_init
-    @requires_process_group_agent
```

Comment:
If we're removing this decorator, should we also remove the function definition for it?

Reply:
@rohan-varma Updated the PR. I refurbished this decorator to make it useful.